contracts/payloads/IGP89/PayloadIGP89.sol

```diff
@@ -0,0 +1,530 @@
+            VaultConfig memory VAULT_LBTC_USDC = VaultConfig({
+                vault: LBTC_USDC_VAULT,
+                vaultType: VAULT_TYPE.TYPE_1,
+                supplyToken: lBTC_ADDRESS,
+                borrowToken: USDC_ADDRESS,
+                baseWithdrawalLimitInUSD: 10_000, // $10k
+                baseBorrowLimitInUSD: 10_000, // $10k
+                maxBorrowLimitInUSD: 15_000 // $15k
+            });
+
+            setVaultLimits(VAULT_LBTC_USDC); // TYPE_1 => 107
+
+            VAULT_FACTORY.setVaultAuth(LBTC_USDC_VAULT, TEAM_MULTISIG, true);
+        }
+
+        {
+            address LBTC_USDT_VAULT = getVaultAddress(108);
+
+            // [TYPE 1] LBTC/USDT vault
+            VaultConfig memory VAULT_LBTC_USDT = VaultConfig({
+                vault: LBTC_USDT_VAULT,
+                vaultType: VAULT_TYPE.TYPE_1,
+                supplyToken: lBTC_ADDRESS,
+                borrowToken: USDT_ADDRESS,
+                baseWithdrawalLimitInUSD: 10_000, // $10k
+                baseBorrowLimitInUSD: 10_000, // $10k
+                maxBorrowLimitInUSD: 15_000 // $15k
+            });
+
+            setVaultLimits(VAULT_LBTC_USDT); // TYPE_1 => 108
+
+            VAULT_FACTORY.setVaultAuth(LBTC_USDT_VAULT, TEAM_MULTISIG, true);
+        }
+
+        {
+            address LBTC_GHO_VAULT = getVaultAddress(109);
+
+            // [TYPE 1] LBTC/GHO vault
+            VaultConfig memory VAULT_LBTC_GHO = VaultConfig({
+                vault: LBTC_GHO_VAULT,
+                vaultType: VAULT_TYPE.TYPE_1,
+                supplyToken: lBTC_ADDRESS,
+                borrowToken: GHO_ADDRESS,
+                baseWithdrawalLimitInUSD: 10_000, // $10k
+                baseBorrowLimitInUSD: 10_000, // $10k
+                maxBorrowLimitInUSD: 15_000 // $15k
+            });
+
+            setVaultLimits(VAULT_LBTC_GHO); // TYPE_1 => 109
+
+            VAULT_FACTORY.setVaultAuth(LBTC_GHO_VAULT, TEAM_MULTISIG, true);
+        }
+    }
+
+    // @notice Action 2: Set dust limits for RLP-USDC DEX
+    function action2() internal isActionSkippable(2) {
+        address RLP_USDC_DEX = getDexAddress(28);
+
+        // RLP-USDC DEX
+        DexConfig memory DEX_RLP_USDC = DexConfig({
+            dex: RLP_USDC_DEX,
+            tokenA: RLP_ADDRESS,
+            tokenB: USDC_ADDRESS,
+            smartCollateral: true,
+            smartDebt: false,
+            baseWithdrawalLimitInUSD: 10_000, // $10k
+            baseBorrowLimitInUSD: 0, // $0
+            maxBorrowLimitInUSD: 0 // $0
+        });
+        setDexLimits(DEX_RLP_USDC); // Smart Collateral
+
+        DEX_FACTORY.setDexAuth(RLP_USDC_DEX, TEAM_MULTISIG, true);
+    }
+
+    // @notice Action 3: Set dust limits for wstUSR DEXes and vaults
+    function action3() internal isActionSkippable(3) {
+
+        {
+            address wstUSR_USDC_VAULT = getVaultAddress(110);
+
+            // [TYPE 1] wstUSR/USDC vault
+            VaultConfig memory VAULT_wstUSR_USDC = VaultConfig({
```

Comment:
keep this, by comment was on the wrong position before

Reply:
got it